packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
-##41.16.0 - January 8, 2018
+## 41.16.0 - January 8, 2018
 ### Added
-#### salesforce-vscode-apex
-* Update apex-jorje-lsp.jar to enable go to definition for type usage sites ([PR #247] (https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### All
+* Upgrade to TypeScript 2.6.2 ([PR #242] (https://github.com/forcedotcom/salesforcedx-vscode/pull/242))
```

Comment:
The space between the square bracket and the parenthesis prevents the link from working. The spaces should be removed in all of these change log entries.

Reply:
Remove this. It's not necessary and has not customer facing impact.